### ISSUE TITLE
Kindly ask for the discussion and citation of previous works.

### ISSUE THREAD

**Issue by Klayand** [cvlab-kaist/NoiseRefine#1]
Dear authors,

We hope this message finds you well. We recently had the opportunity to read your paper titled “A Noise is Worth Diffusion Guidance,” and we wanted to express our appreciation for your contributions to the field.

While reviewing your work, we noticed several notable similarities with our previously preprint papers titled “Noise Prompt Learning: Learning the Winning Tickets for Diffusion Sampling”, “The Blessing of Smooth Initialization for Video Diffusion Models”, submitted and open as ICLR 2025 submissions. **They are previous works not concurrent works. We kindly hope you can discuss how your work relate to ours.** We summarize the highly relevant points as follows:

1. The noise learning framework appears to be quite similar.
In your work, you aim to optimize the noise by calculating the MSE loss between the original noise and the predicted noise.
![image](https://github.com/user-attachments/assets/cbeda8b3-d069-44bc-8c27-b8fab5c866f9)

**(Zikai Zhou's work)** In Noise Prompt Learning, we are the first to formulate the noise learning framework, including the noise pairs collection, noise prediction model architecture, and inference with the noise prediction model. We also calculate the MSE loss between the original noise and the predicted noise.
![image](https://github.com/user-attachments/assets/816713d0-2c82-4317-a43c-b9c728289cff)
![image](https://github.com/user-attachments/assets/d14525e9-c58d-4fc2-b86c-25b519442428)

2. The dataset collection methodology is comparable.
In your work, you collect the inversion noises by utilizing the DDIM Inversion, which is the same as in ”Noise Prompt Learning: Learning the Winning Tickets for Diffusion Sampling”.
![image](https://github.com/user-attachments/assets/633f545f-a243-4ffc-974a-b141d3859df0)
![image](https://github.com/user-attachments/assets/60842daa-b28e-48a0-80f9-9d4fb62a239b)

**(Zikai Zhou's work)** In Noise Prompt Learning, we utilize the DDIM Inversion to obtain the noise pairs from x_t -> x_{t-1} - > x_t^\prime. The formulation is shown in below:
![image](https://github.com/user-attachments/assets/a4a2a81d-fc99-471c-ae85-e01dd81ae1f3)
![image](https://github.com/user-attachments/assets/4ff51412-e76f-49ad-95bb-460dc9e7974b)

3. There are parallels in the design of the model architecture and training approach.
In your work, you design the noise refine LORA, and use it to predict the residual between the input noise and the inversion noise, similar to the residual prediction branch in ”Noise Prompt Learning: Learning the Winning Tickets for Diffusion Sampling”. 
![image](https://github.com/user-attachments/assets/d0500817-f38b-46a9-b288-96c96f01e657)

**(Zikai Zhou's work)** In Noise Prompt Learning, we design two branches: one is the singular value prediction, another is the residual prediction. For the residual prediction, we aim to predict the residuals between the refined noise. 
![image](https://github.com/user-attachments/assets/c0ca2a1e-a85d-43c5-a048-f48ea3d54845)

4. The theoretical proofs share a resemblance.
The Lipschitz constant derivation: The error at time t-1 can be expressed as the difference between the error at time t and the result after passing through the model (i.e., the derivative).
![image](https://github.com/user-attachments/assets/d0e0bcab-4379-4b0a-8789-59f974c95376)

**(Shitong Shao's work)** In SmoothInit: After removing the norm and multiplying both sides by \(\Delta x_T\), and restoring the first-order Taylor expansion to "the difference of the result after passing through the model," the conclusion is the same as above.
![image](https://github.com/user-attachments/assets/3e3504d0-605d-468a-8d36-6f28dafbd5b2)
The bound derivation: you obtain the interpolation of \(x_0\) through recursive operations, which can be bounded by the interpolation of \(x_T\).
![image](https://github.com/user-attachments/assets/5d07b341-738f-4437-98df-08dffe845bd1)

**(Shitong Shao's work)** In SmoothInit: We also use recursion to arrive at a conclusion that is almost identical. However, we additionally prove that the value of \(Z(0)\) is the maximum, thus choosing this tightest bound.
![image](https://github.com/user-attachments/assets/484a063f-ef6b-4c71-93ce-a2828871248c)

In fact, this proof had preliminary conclusions in Shitong Shao's 2023 work on flow matching, and they did not cite it. (**Shitong Shao's work, Precise Knowledge Transfer via Flow Matching)**, in flow matching, \(\alpha\) and \(\sigma\) are respectively \(t\) and \(1-t\), so they can be omitted:
![image](https://github.com/user-attachments/assets/7fae20e1-ff9b-4f45-8720-39848cda48d2)

Given these similarities, we are surprised to see that our works are not discussed in your paper. We believe that proper citation is important for acknowledging previous research. We also hope you can discuss the relationship between ours and your work in your paper.

If you are open to it, we would appreciate the opportunity to discuss this matter further. we think it would be beneficial for us to clarify our respective contributions and ensure that both of our works are appropriately recognized in the academic community.

Thank you very much for your time and consideration. we look forward to your response and hope we can connect soon.

Warm regards,

Zikai Zhou

@misc{zhou2024goldennoisediffusionmodels,
      title={Golden Noise for Diffusion Models: A Learning Framework}, 
      author={Zikai Zhou and Shitong Shao and Lichen Bai and Zhiqiang Xu and Bo Han and Zeke Xie},
      year={2024},
      eprint={2411.09502},
      archivePrefix={arXiv},
      primaryClass={cs.LG},
      url={https://arxiv.org/abs/2411.09502}, 
}

@misc{
zhou2024noise,
title={Noise Prompt Learning: Learning the Winning Tickets for Diffusion Sampling},
author={zikai zhou and Shitong Shao and Bai LiChen and zhiqiang xu and Bo Han and Zeke Xie},
year={2024},
url={https://openreview.net/forum?id=qIJenSdGbW}
}

@misc{
shao2024the,
title={The Blessing of Smooth Initialization for Video Diffusion Models},
author={Shitong Shao and Bai LiChen and zikai zhou and Tian Ye and Yunfeng Cai and Kaishun Wu and Zeke Xie},
year={2024},
url={https://openreview.net/forum?id=QIp1YUqgCB}
}

@article{shao2024precise,
  title={Precise Knowledge Transfer via Flow Matching},
  author={Shao, Shitong and Shen, Zhiqiang and Gong, Linrui and Chen, Huanran and Dai, Xu},
  journal={arXiv preprint arXiv:2402.02012},
  year={2024}
}

**Comment by sunovivid** [comment]
Dear Zikai Zhou,

Thank you for your interest in our work and for bringing these relevant studies to our attention. We appreciate your time and effort to review our paper and share your insights.

We have carefully reviewed the papers you mentioned and find them to be fascinating contributions to the field. **Following your suggestions, we will appropriately cite your work in our paper.** However, it seems there might be some misleadingness about our work. Below, we aim to clarify these points and highlight the key differences between Noise Prompt Learning and our study.

### Regarding the misleading

**We would like to clarify that our training framework does not involve building a dataset of inversion and predicted noise pairs and optimizing an MSE loss between them.**

From your description:

> In your work, you design the noise refine LORA, and use it to predict the residual between the input noise and the ***inversion noise**, similar to the residual prediction branch in ”Noise Prompt Learning: Learning the Winning Tickets for Diffusion Sampling”*
> 

and

> you aim to optimize the noise by calculating the MSE loss between the original noise and the predicted noise.
> 

It seems that our approach might have been misleading. In our work, we calculate the loss as a distance metric in the image space, not directly in the noise space, and do not perform any inversion in our training framework.

This approach is necessary because performing full-step inversions accumulates approximation errors, making it challenging to obtain exact inversion noise. Consequently, instead of relying on inversions, we calculate the loss based on the images produced by denoising the refined noise. Therefore, our training framework does not rely on a dataset of noise pairs.

> In your work, you collect the inversion noises by utilizing the DDIM Inversion
> 

You seem to be referring to the motivational experiments in our work, which were designed to explore the relationship between Gaussian noise and guidance-free noise. However, even here, there are significant differences, which we will discuss more below.

### Difference between Noise Prompt Learning

First, the problems we aim to address are distinct. While your paper focuses on achieving better text-image alignment and improving human preference metrics, our work centers on _replacing the role of guidance_ in diffusion models with noise prompts. Considering the significant role noise plays in determining the final image [1, 2], refining the noise to replicate the function of guidance was a natural direction for exploration in our study. As a result, our method does not require CFG in inference time, while Noise Prompt Learning still requires CFG in inference time.

Second, in the motivational study (Sec. 3.1), we perform _full-step_ denoising and inversion, whereas Noise Prompt Learning perform one-step re-denoising. When aiming to construct 'guidance-free' noise, generating a plausible image with guidance and then obtaining its inversion noise is a logical and straightforward approach. Our methodologies also diverge due to the differing goals (better prompt alignment & human preference vs guidance-free generation). In Noise Prompt Learning, one-step de-noising is sufficient, whereas our method requires full-step de-noising. Since CFG affects the generated image across multiple steps of the denoising process to generate high-quality images, applying one-step re-denoising would not achieve our intended objectives.

Third, we calculate distance loss in _image space_, not noise space, due to the error accumulation in the inversion process. Calculating the loss in the image space instead of the noise space leads to additional key differences. For instance, retaining gradients across multiple denoising steps would result in significant computational overhead, which is infeasible in most GPUs. To address this, we proposed a novel method called Multistep Score Distillation (MSD) to avoid this cost. Further, using an image-space loss opens up opportunities to incorporate other loss functions, such as GAN or perceptual losses.

While there are differences between our works, we agree that your work should be appropriately recognized as it contributes to the framework of learning noise for diffusion models. We will make sure we cite your work appropriately in our paper.

Thank you again for your interest in our work and your valuable feedback. If you have further concerns or would like to discuss this matter further, please do not hesitate to reach out 🥰

Warm regards,

Donghoon Ahn, Jiwon Kang

[1]: Dvir Samuel, Rami Ben-Ari, Simon Raviv, Nir Darshan, and Gal Chechik. Generating images of rare concepts using pre-trained diffusion models. In Proceedings of the AAAI Conference on Artificial Intelligence, pages 4695–4703, 2024.

[2]: Katherine Xu, Lingzhi Zhang, and Jianbo Shi. Good seed makes a good crop: Discovering secret seeds in text-to-image diffusion models. arXiv preprint arXiv:2405.14828, 2024.

**Comment by Klayand** [comment]
Dear sunovivid,

T**hank you for your response and for your willingness to include relevant discussions in future versions of your work**. We appreciate your efforts to contribute to the community and advance research in this field.

We will pay close attention to the relevant discussion in your future version and look forward to your interesting contribution to the community. We look forward to future opportunities for collaboration and engaging discussions.

Zikai Zhou

**Comment by Klayand** [comment]
**We hope you will highlight both the similarities and differences between our approaches in your discussions.** The readers and community will judge the novelty and contribution of your work.

It has been a pleasure to connect with you, and we are excited to see how your work will further contribute to advancing this direction in our field.

**Comment by sunovivid** [comment]
Thank you for your feedback. We appreciate your time and will make every effort to appropriately reflect your contributions and clarify the similarities and differences between our works in future revisions. Thanks again for your exciting work, and we look forward to seeing your subsequent research. 😊
